src/Common/Configuration.ts

```diff
@@ -0,0 +1,44 @@
+﻿import { config } from "mssql";
```

Comment:
nit: can you use `config as SqlConfig` to disambiguate it?

Reply:
Removed it.